### DISCUSSION TITLE
Random questions on TE3

### DISCUSSION THREAD

**Discussion by ericdaofin** [TabularEditor/TabularEditor3#235] [🙋♀️TE3 Q&A] [answered]
Hi, just started to use TE3 today and I really like it. I have a few random questions below, hope someone can answer please. Thank you in advance.

- Duplicate measure in TE2. Now in TE3 I have to copy and paste a measure?
- Can I edit a relationship in TE that connects to Power BI Desktop? The "From column" and "To column" are greyed out.
- In the Model menu on the Menu bar, some features are greyed out such as "Create New Calculated Table", "New Relationship" etc. How can I enable these features?

I am using Personal License.

Thanks again.

**Comment by otykier** [comment, marked as answer]
> Duplicate measure in TE2. Now in TE3 I have to copy and paste a measure?

Yes for now, although several people have asked to get the Duplicate option back, so I will include it in a future update.

> Can I edit a relationship in TE that connects to Power BI Desktop? The "From column" and "To column" are greyed out.

This is [unsupported by Power BI Desktop](https://docs.microsoft.com/en-us/power-bi/transform-model/desktop-external-tools#supported-write-operations). You can experiment by enabling unsupported modeling features under **Tools > Preferences > Power BI**.

> In the Model menu on the Menu bar, some features are greyed out such as "Create New Calculated Table", "New Relationship" etc. How can I enable these features?

Same answer as above.

**Reply by ericdaofin** [reply]
Thank you. It works!
